docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM nvidia/cuda:11.2.2-cudnn8-runtime-ubuntu20.04
+LABEL maintainer="Hugging Face"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt update
+RUN apt install -y git libsndfile1-dev tesseract-ocr espeak-ng python3 python3-pip ffmpeg
+RUN python3 -m pip install --no-cache-dir --upgrade pip
+
+ARG REF=master
+RUN git clone https://github.com/huggingface/transformers && cd transformers && git checkout $REF
+RUN python3 -m pip install --no-cache-dir -e ./transformers[dev,onnxruntime]
+
+RUN python3 -m pip install --no-cache-dir -U torch tensorflow
+RUN python3 -m pip uninstall -y flax jax
```

Comment:
flax  jax is not included yet no? => sounds good to me! 

Don't we need https://github.com/huggingface/transformers/blob/86119c115496ca773b82f8a1c8cbf2d4e44149fa/.github/workflows/self-scheduled.yml#L39 here as well though?

Reply:
It's two lines below Patrick ;-)

Reply:
indeed thanks!